net-im/slack++/slack++-0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Enterprise chat and services on the command line - slack++"
+DESCRIPTION="slack++ is an ncurses based client the Searchable Log of All Conversation and Knowledge"
+HOMEPAGE="https://github.com/ga2arch/slack"
+COPYRIGHT="2018"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="961dec741764dfb81999703b84989d93ec7a88ee"
+SOURCE_URI="https://github.com/ga2arch/slack/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9c84b5d0ee7b528edfdde219263dd7c307c6e8cd9aadc491e7148f7457ea30a5"
+SOURCE_DIR="slack-$srcGitRev"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
Insert an emtpy line here, and also
```diff
-SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
```
no need to call if it doesn't build for x86_gcc2 :)

Reply:
Yeah but isn't there technically supposed to eventually be x86_gcc2 on x86_64?

Reply:
If it doesn't build on primaryArch it won't either on secondaryArch :) "!x86_gcc2"

Reply:
The rule is a bit special: We do mention broken arches in `ARCHITECTURES`, but we prefer not to mention them in `SECONDARY_ARCHITECTURES`.

Reply:
Ok this makes sense now I think.